src/analysis/inner_product_space/spectrum.lean

```diff
@@ -244,3 +244,23 @@ end version2
 
 end is_self_adjoint
 end inner_product_space
+
+section nonneg
+
+lemma eigenvalue_nonneg_of_nonneg {μ : 𝕜} {T : E →ₗ[𝕜] E} (hμ : module.End.has_eigenvalue T μ)
+  (hnn : ∀ (x : E), 0 ≤ is_R_or_C.re ⟪T x, x⟫) : 0 ≤ is_R_or_C.re μ :=
```

Comment:
I just remembered, mathlib's inner products are conjugate linear in the *first* coordinate, not the second (I am used to the latter), so it makes more sense to apply `T` in the second coordinate.
```suggestion
  (hnn : ∀ (x : E), 0 ≤ is_R_or_C.re ⟪x, T x⟫) : 0 ≤ μ :=
```

Reply:
This seems reasonable to me, too.